index.html

```diff
@@ -237,7 +237,7 @@
       <p>The appropriate times to connect to a host, or <a href="http://www.w3.org/TR/html5/document-metadata.html#concept-link-obtain">obtain</a> the specified resource are:</p>
 
       <ul>
-        <li>When the user agent that supports [[!RFC5988]] processes <a>resource hint link</a>'s specified via the <a href="http://tools.ietf.org/html/rfc5988">Link HTTP header</a>.</li>
+        <li>When the user agent that supports [[!RFC5988]] processes <a>resource hint link</a>s specified via the <a href="http://tools.ietf.org/html/rfc5988">Link HTTP header</a>.</li>
```

Comment:
For consistency, we should probably make that singular? "... processes a resource hint link specified via"